Pop-Grupa-A-Frontend/src/components/computationCockpit/applicationList/AppList.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+
+class AppList extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      chosenAppId: -1,
+      userId: 1,
+    };
+    this.createTask = this.createTask.bind(this);
+  }
+
+  onAppClick = (chosenAppId) => {
+    this.setState({ chosenAppId });
+  }
+
+  createTask(e) {
```

Comment:
przekazywanie eventu i wywoływanie metody `event.preventDefault()` chyba nie jest tu konieczne.
To jest zwykły przycisk ( nie submit), więc nic się nie dzieje nadzwyczajnego.
Czy się może mylę?